apache-maven/src/assembly/maven/conf/maven.properties

```diff
@@ -0,0 +1,20 @@
+#
+# Maven user properties
+#
+
+maven.user.home = ${user.home}/.m2
```

Comment:
Based on my previous comment:  `maven.user.conf`? Akin to `maven.conf`?

Reply:
for ref (and ideas): https://github.com/maveniverse/mima/blob/main/context/src/main/java/eu/maveniverse/maven/mima/context/MavenUserHome.java

Meaning, I am 100% with value key name.

Reply:
That wouldn't be consistent with the global props. All of those XML files are config files, at the end.

Reply:
I disagree, as that IS "maven user home", so "user home" plus something, local repo is not config, is it?

Reply:
Plus, that is where mvnd daemons logs, where indexer put some files, and so on... it is IMHO really "maven user home".

Reply:
> I disagree, as that IS "maven user home", so "user home" plus something, local repo is not config, is it?

It is not, and this is the reason why it has its own property to be relocated. For technical reasons we can only assume the user home to be writable, but that can be a problem if it is NFS mounted or similar.

Reply:
> Plus, that is where mvnd daemons logs, where indexer put some files, resolver put file locks, and so on... it is IMHO really "maven user home".

Then we need `maven.user.workdir` or similar.

Reply:
@cstamas The problem you describe has been solved by Windows with `APPDATA\{Roaming,Local,LocalLow}` the the conf files will roam, but the generated stuff will be in `Local`, if not even `LocalLow`, thus never be network mounted.